app/platform/config.go

```diff
@@ -36,13 +36,17 @@ func (c *ServiceConfig) validate() error {
 
 	if c.Logger == nil {
 		var err error
-		// If Logger is not set, use a default logger temporarily.
```

Comment:
Doesn't this mean that now we can remove this logger initialization?

Reply:
This is pretty much the default behavior in server in the old `s.InitLogging()` method. If logging is not configured, it just creates a default logger.

Reply:
Essentially, we aren't using the `Logger` field from `ServiceConfig` at all. It is always initialized from scratch. So this condition will always succeed. That's why I was thinking we can remove this and directly initialize the logger as we are doing in `initLogging`.

Reply:
Oh I see, yeah the check is not necessary.